src/api/utils.js

```diff
@@ -168,7 +168,10 @@ export const parseUserSaveData = (
     // This is only required when editing users, because new users can't have such properties
     if (!isNewUser) {
         for (const ownedPropName of userModelOwnedProperties) {
-            if (user[ownedPropName] && !data[ownedPropName]) {
+            if (
+                user[ownedPropName] &&
+                typeof data[ownedPropName] === 'undefined'
```

Comment:
I don't know what data contains exactly, if data has this shape:

```js
const data = {
  field: undefined,
  fieldAlso: 'defined'
}
```

I'd expect the current check, but if it has this shape:

```js
const data = {
  fielder: 'defined'
}

field in data // false
```

I can imagine that `ownedPropName in data` would be appropriate, as then you'd be checking whether the prop is in data, instead of having to check the value.

Reply:
On checking this in the app, it does look like the second scenario to me (so using the `in` operator for the check here seems appropriate to me).